Podcast/MiniPlayerView.swift

```diff
@@ -10,10 +10,10 @@ class MiniPlayerView: UIView {
     
     let miniPlayerHeight: CGFloat = 60.5
     let marginSpacing: CGFloat = 17
-    let buttonSize: CGSize = CGSize(width: 15, height: 18)
+    let buttonSize: CGSize = CGSize(width: 20, height: 24)
```

Comment:
this actually looks a bit wonky on my phone in terms of size ... make it a tiny bit smaller